Tests/fixtures/clienthints.yml

```diff
@@ -77,3 +77,146 @@
     model: ZenFone 4 Max
   os_family: Android
   browser_family: Opera
+-
+  user_agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/97.0.4692.99 Safari/537.3
+  headers:
+    Sec-CH-UA: '" Not A;Brand";v="99", "Chromium";v="97", "Google Chrome";v="97"'
+    Sec-CH-UA-Mobile: "?0"
+    Sec-CH-UA-Platform: "Windows"
+    Sec-CH-UA-Platform-Version: "14.0.0"
+  os:
+    name: Windows
+    version: "11"
+    platform: "x64"
+  client:
+    type: browser
+    name: Chrome
+    version: "97.0.4692.99"
+    engine: Blink
+    engine_version: ""
+  device:
+    type: desktop
+    brand:
+    model:
+  os_family: Windows
+  browser_family: Chrome
+-
+  user_agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/97.0.4692.99 Safari/537.3
+  headers:
+    Sec-CH-UA: '" Not A;Brand";v="99", "Chromium";v="97", "Google Chrome";v="97"'
+    Sec-CH-UA-Mobile: "?0"
+    Sec-CH-UA-Platform: "Windows"
+  os:
+    name: Windows
+    version: "10"
+    platform: "x64"
+  client:
+    type: browser
+    name: Chrome
+    version: "97.0.4692.99"
+    engine: Blink
+    engine_version: ""
+  device:
+    type: desktop
+    brand:
+    model:
+  os_family: Windows
+  browser_family: Chrome
+-
+  user_agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36 Edg/95.0.1020.44
+  headers:
+    Sec-CH-UA: '" Not A;Brand";v="99", "Chromium";v="95", "Microsoft Edge";v="95"'
+    Sec-CH-UA-Mobile: "?0"
+    Sec-CH-UA-Platform: "Windows"
+    Sec-CH-UA-Platform-Version: "14.0.0"
+  os:
+    name: Windows
+    version: "11"
+    platform: "x64"
+  client:
+    type: browser
+    name: Microsoft Edge
+    version: "95.0.1020.44"
+    engine: Blink
+    engine_version: ""
+  device:
+    type: desktop
+    brand:
+    model:
+  os_family: Windows
+  browser_family: Internet Explorer
+-
+  user_agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.82 Safari/537.36 OPR/84.0.4316.19
+  headers:
+    Sec-CH-UA: "\" Not A;Brand\";v=\"99\", \"Chromium\";v=\"98\", \"Opera\";v=\"84\""
+    Sec-CH-UA-Mobile: "?0"
+    Sec-CH-UA-Full-Version-List: "\" Not A;Brand\";v=\"99.0.0.0\", \"Chromium\";v=\"98.0.4758.82\", \"Opera\";v=\"98.0.4758.82\""
```

Comment:
Are the Opera versions really this different?

Reply:
I can also confirm from the data I have, that the versions are different.